tests/PKSim.Tests/Presentation/SimulationComparisonTreeNodeContextMenuFactorySpecs.cs

```diff
@@ -20,7 +20,7 @@ protected override void Context()
    public class When_checking_if_the_simulation_comparison_tree_node_context_menu_factory_can_create_a_context_menu_for_a_given_node : concern_for_SimulationComparisonTreeNodeContextMenuFactory
    {
       [Observation]
-      public void shouldu_return_true_if_the_node_is_a_classifiable_comparision_node()
+      public void should_return_true_if_the_node_is_a_classifiable_comparision_node()
```

Comment:
still a typo lol comparision => comaprison

Reply:
There are a few instances of that. I'll fix them all, but one is in a database query that I won't change.

```
migrateTo5_3
```